### ISSUE TITLE
Bump search terms query to find at minimum 5 stars

### ISSUE THREAD

**Issue by mrthankyou** [JLLeitschuh/lgtm_hack_scripts#15]
I've noticed when I run a query via a search term, I will get A LOT of projects that don't have any stars. This causes the scripts to run at an unreasonable amount of time. In testing I've become accustomed to adding `stars:>5` to the search terms query script in order to get relevant repos while at the same time making sure my script won't run forever.

*Question*: Does it make sense for us to make this a permanent change and add `stars:>5` to to the search terms query?

**Comment by JLLeitschuh** [comment]
Yes. That seems completely reasonable. In a majority of cases, I probably won't report a vulnerability I uncover if the project has under 100 unless the project is owned by an organization with a BB program.

**Comment by mrthankyou** [comment]
Good to note. If you want to move it to 100 stars that works for me too. You have the experience here so I'll defer to you.

**Comment by JLLeitschuh** [comment]
I wish there was a way to say 100 stars, except for these orgs with BB programs, then the limit is 5. Can we do something like that?
